docs/conf.py

```diff
@@ -3,10 +3,10 @@
 import sys
 from os import path
 
-version = ''
+
 # Centralize environment variables
 JINA_VERSION = os.environ.get('JINA_VERSION')
-SPHINX_MULTIVERSION_VERSION = os.environ.get('SPHINX_MULTIVERSION_VERSION', version)
+SPHINX_MULTIVERSION_VERSION = os.environ.get('SPHINX_MULTIVERSION_VERSION', None)
```

Comment:
no, this is not correct, this assignment needs to go much lowrr, aftrr the version variable is properly set

Reply:
> no, this is not correct, this assignment needs to go much lowrr, aftrr the version variable is properly set

Oh. I am extremely sorry I didn't notice that...

Reply:
Is that fine?
![kolla](https://github.com/jina-ai/jina/assets/68547750/b3219c42-657d-481d-ac2a-b03c5a508f7c)

Reply:
no. Follow the flow, in master code we have:

```python
os.environ.get("SPHINX_MULTIVERSION_VERSION", version) on line 170
```
and on line 31 we have:

```python
version = __version__ 
```

With your changes, the default would be empty, in master the default would be the value of `__version__`. So ur code does not refactor, but changes the flow of the code

Reply:
> no. Follow the flow, in master code we have:
> 
> ```python
> os.environ.get("SPHINX_MULTIVERSION_VERSION", version) on line 170
> ```
> 
> and on line 31 we have:
> 
> ```python
> version = __version__ 
> ```
> 
> With your changes, the default would be empty, in master the default would be the value of `__version__`. So ur code does not refactor, but changes the flow of the code


Should I add a empty `version` string at the begining?
![Screenshot_6](https://github.com/jina-ai/jina/assets/68547750/d3aafb03-243b-4597-a715-a6c6c965d132)

Reply:
no, there is no need. But you cannot do this 

`os.environ.get("SPHINX_MULTIVERSION_VERSION", version)`

So early in the code

Reply:
the code 

```python
SPHINX_MULTIVERSION_VERSION  = os.environ.get('SPHINX_MULTIVERSION_VERSION', version)
```
needs to be after line 31

Reply:
yes

Reply:
> 

Ok